content/en/code_analysis/github_pull_requests.md

```diff
@@ -14,17 +14,18 @@ further_reading:
 
 ## Overview
 
-Code Analysis can automatically flag violations in pull requests on GitHub. To configure the [GitHub integration][2] to include Code Analysis features, see [the setup instructions](#set-up-code-analysis-for-github-pull-requests).
-
-During code reviews on GitHub, the integration checks for Static Analysis violations in pull requests for repos that have at least one ruleset applied. Violations are flagged with a review comment on the relevant line of code.
-
+Code Analysis integrates with GitHub pull requests in two ways:
+- [Pull request comments to flag violations][7]: During code reviews on GitHub, Datadog can automatically check for Static Analysis violations in pull requests for repos that have at least one ruleset applied. Violations are flagged with an in-line review comment on the relevant line(s) of code, along with suggested fixes (when applicable) that can be applied directly in the pull request. This is only available for Static Analysis (SAST).
```

Comment:
I would think `inline` fits better here rather than `in-line` (which has a different meaning imo), but I could be wrong.

```suggestion
- [Pull request comments to flag violations][7]: During code reviews on GitHub, Datadog can automatically check for Static Analysis violations in pull requests for repos that have at least one ruleset applied. Violations are flagged with an inline review comment on the relevant line(s) of code, along with suggested fixes (when applicable) that can be applied directly in the pull request. This is only available for Static Analysis (SAST).
```